Library/Homebrew/requirements/java_requirement.rb

```diff
@@ -42,4 +42,12 @@ def message
   def inspect
     "#<#{self.class.name}: #{name.inspect} #{tags.inspect} version=#{@version.inspect}>"
   end
+
+  def display_s
+    if @version
+      "#{name} = #{@version}"
```

Comment:
`>` rather than `=`, no? Also this should probably differentiate between `1.7` and `1.7+`